PAS/Management/OEManager/ABL/bin/oemanager.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
What was the reason you switched from the default shell to the bash shell?

Reply:
Covered in the ticket. Mostly due to the product finally being "fixed" for Ubuntu in 12.8 using the same approach. This keeps pace with that as the official fix.